x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/ConnectorIndexServiceTests.java

```diff
@@ -295,6 +296,24 @@ public void testUpdateConnectorNative() throws Exception {
         assertThat(isNative, equalTo(indexedConnector.isNative()));
     }
 
+    public void testUpdateConnectorStatus() throws Exception {
```

Comment:
Do you want to add a test case where an invalid new status is passed

Reply:
Good point, I will add the unit test :)